tools/docker-compose-cluster/awx-1-receptor-standalone.conf

```diff
@@ -0,0 +1,25 @@
+---
+- node:
+    id: receptor_1
+
+- log-level: info
+
+#- tls-client:
+#    name: mutual-tls-client
+#    cert: /etc/receptor/certs/receptor_1.crt
+#    key: /etc/receptor/certs/receptor_1.key
+#    rootcas: /etc/receptor/certs/ca.crt
+
+- tcp-peer:
+    address: awx_1:4444
+    redial: true
+    #tls: mutual-tls-client
+
+- work-command:
+    worktype: ansible-runner
+    command: ansible-runner
+    params: worker
+    allowruntimeparams: true
+
+- control-service:
+    service: control
```

Comment:
This file should definitely not go in the `tools/docker-compose-cluster` folder, because it's for docker-compose, which has its own folder.

If we're moving it anyway, it might also be good to name it something other than "receptor". We have historically called these "receptor nodes", but in planning we veered away from that, and "execution nodes" became more of the nomenclature. I would like a little more clarity on the language we use, but that's more the responsibility of the people working on adding the `node_type` field.

So we'll have 1 control node and 1 execution node. This is the receptor.conf for the execution node. The control node has the `tools/docker-compose/receptor.conf` file, which is statically ADDed to the Dockerfile. I don't know if that makes sense. Clearly the control node conf file won't have `tcp-peer`, but it seems like it doesn't need that?